packages/devtools_app/lib/src/screens/memory/panes/leaks/instrumentation/model.dart

```diff
@@ -81,27 +81,30 @@ class LeakReport {
     required this.type,
     required this.details,
     required this.code,
+    required this.disposalStack,
   });
 
   factory LeakReport.fromJson(Map<String, dynamic> json) => LeakReport(
         type: json['type'],
         details: (json['details'] as List<dynamic>? ?? []).cast<String>(),
         code: json['code'],
+        disposalStack: json['disposalStack'],
```

Comment:
The strings are not used in any other places. What's the point of creating the constants?

Reply:
they are used in the toJson method below. Placing strings in consts is a practice we follow across DevTools and has the benefits:
1) the variable name and / or dart doc can offer insights about what the string is for beyond what the string is itself (much like we prefer to use constants for "magic" or "special" numbers where those exist)
2) placing the string in a const will enable reuse of the string in the future if the need arises, and is more likely to prevent typo bugs from creeping in
3) general consistency across the codebase